src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,194 @@
+package duke;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+/**
+ Class to process the Tasklist of the chatbot
+ */
+public class TaskList {
+
+    /**
+     * Prints the arraylist for the user of the chatbot
+     *
+     * @param storeList arraylist to be printed
+     * @return Nothing
+     *
+     */
+    public static void printTheList(ArrayList<Task> storeList) {
```

Comment:
Method names can follow a standardised convention. For example, listCommand, deleteTaskCommand, unmarkCommand